sysutils/lima/Portfile

```diff
@@ -28,34 +28,31 @@ maintainers         {gmail.com:herby.gillot @herbygillot} \
                     {macports.halostatue.ca:austin @halostatue} \
                     openmaintainer
 
-checksums           rmd160  3ec50fca233273c7872ecf19d8a22836e9b573c2 \
-                    sha256  0c5e601c3788eaaa9f45bdf00d69e23b5720dd2f35e1be36ad21c032fdabf738 \
-                    size    7413280
+checksums           rmd160  537cdb940ebbd891c34af018eb191b3426171fb5 \
+                    sha256  82577e2223dc0ba06ea5aac539ab836e1724cdd0440d800e47c8b7d0f23d7de5 \
+                    size    7414060
 
 build.cmd           make
 
-build.args-append   native
-
 patchfiles          patch-Makefile.diff
 
 platform darwin {
     # Lima defaults to VZ with macOS 13.5 and later; drop dependency from 14 onwards
     if {${os.major} < 23} {
-        default_variants +additional_guestagents
+        depends_run-append port:qemu
+    } else {
+        # added January 2025
+        notes {
+            Please note that the Lima now defaults to native\
+            virtualization support and not QEMU. If you rely on it,\
+            such as for emulating other architectures, you can install\
+            it explicitly:
+
+            port install qemu
+        }
     }
 }
 
-variant additional_guestagents description {Guest agents for all architectures} {
-  build.args-append  additional-guestagents
-  depends_run-append port:qemu
-}
```

Comment:
Why removed `additional_guestagents`?

Reply:
This was unintentional. I'll revise.

Reply:
https://github.com/macports/macports-ports/pull/28513